backend/include/sdk.hpp

```diff
@@ -119,6 +119,7 @@ class SDK : public std::enable_shared_from_this<SDK> {
 
     void publishStationState(const nlohmann::json& state, bool broadcastToElectron = true);
     void publishMainVolumeChange(const float& volume, bool broadcastToElectron = true);
-    void publishStationAdded(const std::string& callsign, const int& frequencyHz);
+    void publishStationAdded(const std::string& callsign, const int& frequencyHz,
+        const std::optional<int>& frequencyAlias = std::nullopt);
```

Comment:
Optional because the `AddFrequency()` method also calls this function and doesn't have a `frequencyAlias` to provide.